utils/fuctions/content/image.py

```diff
@@ -0,0 +1,91 @@
+from urllib.parse import urlparse, parse_qs, urlencode, urlunparse
+import os
+import requests
+import re
+
+def update_url_type_param(url, new_type_value): # url에서 type 파라미터 값을 변경
```

Comment:
문제: 디렉토리 이름이 명확하지 않은 것 같아요
아래와 같이 수정해보면 어떨까요?

```suggestion
utils/functions/content > utils/functions/blog_content
utils/functions/content > utils/functions/blog_meta
```